apps/concierge_site/lib/mailchimp.ex

```diff
@@ -1,96 +1,84 @@
 defmodule ConciergeSite.Mailchimp do
   @moduledoc """
-  functions for sending unsubscribes and subscribes
+  Handles updating digest subscribership in Mailchimp and receiving updates from the service.
   """
 
   require Logger
   alias AlertProcessor.Helpers.ConfigHelper
   alias AlertProcessor.Model.User
   alias AlertProcessor.Repo
-  alias HTTPoison
+  alias Ecto.Changeset
+
+  @doc "Update a user's subscriber status in Mailchimp to reflect their `digest_opt_in` value."
+  @spec update_member(User.t()) :: :ok | :error
+  def update_member(%{id: id, email: email, digest_opt_in: digest_opt_in}) do
```

Comment:
I wonder if the method was called something like `upsert_member` if that might help it make sense when we use it for both adding and updating.